app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,64 @@
+class TweetsController < ApplicationController
+
+  get '/tweets/new' do
+    if !logged_in?
+      redirect '/login'
+    end
+    erb :'/tweets/create_tweet'
+  end
+
+  get '/tweets' do
+    if !logged_in?
+      redirect '/login'
+    else
+      @user = current_user
+      erb :'/tweets/tweets'
+    end
+  end
+
+  post '/tweets' do
+    if !params[:tweet][:content].empty?
+      @tweet = Tweet.create(params[:tweet])
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```